govc/importx/ovf.go

```diff
@@ -164,6 +164,21 @@ func (cmd *ovfx) Map(op []Property) (p []types.KeyValue) {
 	return
 }
 
+func (cmd *ovfx) validateNetwork(e *ovf.Envelope, net Network) {
+	var names []string
+
+	if e.Network != nil {
+		for _, n := range e.Network.Networks {
+			if n.Name == net.Name {
+				return
```

Comment:
Is the following scenario possible?

- `e.Network.Networks` contains multiple elements, e.g. "net-1", "net-2"
- `net` is "net-2"
- the first validation (171) would fail and append to `names`
- the second validation (in loop 171) succeeds and the code returns

In this case we don't want to print the warning (179), correct? Only if there's no match found at all?

Reply:
You just wrote the unit test :D Thx for clarification